bin/add-feed.sh

```diff
@@ -0,0 +1,45 @@
+podcast_id=$1
+base_directory="../feeds"
+
+cd "$base_directory" || exit
+
+if [ ! -d "$podcast_id" ]; then
+    mkdir "$podcast_id"
+fi
+
+temp_file="$podcast_id/temp.html"
+response=$(curl -s -w '%{http_code}' -o "$temp_file" "https://smotrim.ru/podcast/$podcast_id")
+status=$(echo $response | awk '{print $NF}')
+
+if [ "$status" -ne 200 ]; then
+    echo "Error: server responded with status $status"
+    rm -rf "./$podcast_id"
+    exit 1
+fi
+
+html=$(<"$temp_file")
```

Comment:
Can we use `doc2data` to get JSON object representing a podcast and `jq`  to transform JSON to XML?

Reply:
Implemented